src/Section/TimeDiffChecker/TimeDiffChecker.js

```diff
@@ -0,0 +1,28 @@
+function timeDiffChecker(num) {
+  const seconds = 1;
+  const minute = seconds * 60;
+  const hour = minute * 60;
+  const day = hour * 24;
+  const mon = day * 30;
```

Comment:
[중요도: 중간]이거는 바로 윗 scope로 올리고 UPPER_SNAKE_CASE로 바꾸면 좋겠군요.
```suggestion

const SECOND = 1; // s는 뺐습니다. 복수가 아니니까요!
const MINUTE = SECOND * 60;
const HOUR = MINUTE * 60;
const DAY = HOUR * 24;
const MONTH = DAY * 30; // 정확하게 이름을 적으면 좋습니다, 그리고 월은 30일, 31일, 28일이 있긴 하나 이 부분은 현재 계산하기 어려워서 이렇게 처리했다라고 인지하고 있겠습니다 ㅎㅎ
function timeDiffChecker(num) {
```